infrastructure/lambda/template.yaml

```diff
@@ -1281,7 +1281,7 @@ Resources:
                 - Name: Method
                   Value: GET
                 - Name: ApiName
-                  Value: !Sub ${AWS::StackName}-PublicKBVApiGatewayId
+                  Value: !Sub "kbv-cri-${AWS::StackName}"
```

Comment:
Isn't the stack name `kbv-cri-api-v1`?

Reply:
This also more about consistency https://github.com/govuk-one-login/ipv-cri-kbv-api/blob/main/infrastructure/lambda/template.yaml#L232

Reply:
It's the current [publicKBVApi](https://github.com/govuk-one-login/ipv-cri-kbv-api/blob/main/infrastructure/lambda/template.yaml#L232) name